sources/packages/backend/libs/services/src/workflow/workflow-client.service.ts

```diff
@@ -197,4 +202,33 @@ export class WorkflowClientService {
       // The error is not thrown here, as we are failing silently.
     }
   }
+
+  /**
+   * When an application change request is approved or declined, this method sends a message to the
+   * waiting workflow process.
+   * @param applicationId application id that is used as correlation key for the message.
+   * @param applicationEditStatus application edit status to be sent to the workflow.
+   * @throws error if there is an error while sending the message.
```

Comment:
`@throws` does not make sense since "// The error is not thrown here, as we are failing silently."